cmd/cluster-capi-operator/main.go

```diff
@@ -109,6 +109,11 @@ func main() {
 		controllers.DefaultManagedNamespace,
 		"The namespace where CAPI components will run.",
 	)
+	sourceNamespace := flag.String(
+		"source-namespace",
+		controllers.DefaultMAPIManagedNamespace,
+		"The namespace where MAPI components will run.",
```

Comment:
The code appears to only use it for secrets; are we actually getting other MAPI components with it? If not, the variable name could be a little more descriptive up here.